server/config/auth.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  'facebookAuth': {
+    clientID: '577393235773311', //need to export this data to env file later
```

Comment:
API keys should not be on github
Look into .env files or reference variables on a gitignore file